File: web-app/online-warehouse/src/app/carrier/dto/carrier.dto.ts
import {CarrierTypeEnum} from "./enum/carrier-type.enum";
import {AddressDto} from "../../shared/address/dto/address.dto";

export class CarrierDto {
  id: number;
  address: AddressDto;
  name: string;
  taxNumber: string;
  carrierType: CarrierTypeEnum;
  trusted: boolean;
  driverInfo: string[];
}

File: core/src/main/java/by/itechart/carrier/dto/CarrierDto.java
package by.itechart.carrier.dto;

import by.itechart.carrier.enums.CarrierType;
import by.itechart.common.dto.AddressDto;
import lombok.Data;

import java.util.List;

@Data
public class CarrierDto {
    private Long id;
    private AddressDto address;
    private String name;
    private String taxNumber;
    private CarrierType carrierType;
    private Boolean trusted;
    private List<String> driverInfo;
}
